src/test/librbd/exclusive_lock/test_mock_ReleaseRequest.cc

```diff
@@ -96,7 +96,7 @@ class TestMockExclusiveLockReleaseRequest : public TestMockFixture {
                                int r) {
     if (mock_image_ctx.object_cacher != nullptr) {
       EXPECT_CALL(mock_image_ctx, invalidate_cache(purge, _))
-                    .WillOnce(WithArg<1>(CompleteContext(r, NULL)));
+                    .WillOnce(WithArg<1>(CompleteContext(r, static_cast<ContextWQ*>(NULL)))); 
```

Comment:
it's more C++ish, and cosmetic in this case . and i'd suggest do it in this commit.